conf/modules.config

```diff
@@ -30,7 +30,7 @@ process {
         publishDir = [
             path: { "${params.outdir}/QC/NACHO" },
             mode: params.publish_dir_mode,
-            saveAs: { filename -> filename.indexOf(".png") > 0 ? "png/${filename}" : filename }
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename.indexOf(".png") > 0 ? "png/${filename}" : filename }
```

Comment:
there is also a nextflow function to get the file extension https://www.nextflow.io/docs/latest/script.html#check-file-attributes